### DISCUSSION TITLE
New net6.0 runtime required at every update

### DISCUSSION THREAD

**Discussion by johannestophoj3s** [gitextensions/gitextensions#11374] [General] [answered]
Hi,

I enjoy git extensions and faithfully upgrade whenever I am suggested to do so. But I wonder if it is necessary that git extensions requires me to update the net6.0 runtime every time?

**Comment by vbjay** [comment, marked as answer]
We target the latest runtime at time of build. This provides latest security updates and patches.

**Reply by RussKie** [reply]
https://dotnet.microsoft.com/en-us/download/dotnet/6.0
![image](https://github.com/gitextensions/gitextensions/assets/4403806/4ad722fe-c3c9-478b-8a9f-2a3644e891da)
